### DISCUSSION TITLE
i need help starting the bot

### DISCUSSION THREAD

**Discussion by Q8tys** [any-other-guy/Endless-Chaos-LostArk#35] [Q&A]
Hello guys,  i have downloaded the Prerequisites and the file for the bot … also I have set the game settings as required … now I open python and copy paste the start running script but i get an syntaxerror: invalid syntax no idea why

**Comment by SCCode-War** [comment]
Hello, i need a guide on how to start the bot too. i'm very new to github and never done any coding so it's hard to understand how it works. Any advise would be much appreciated. if i could message you directly that would be better.

**Comment by zaicanu21** [comment]
need help with the boot discord SuicideSquadRo#9683
